x-pack/plugins/observability_solution/infra/public/apps/legacy_app.tsx

```diff
@@ -27,72 +32,67 @@ export async function renderApp({ element }: AppMountParameters) {
 
 const LegacyApp: React.FunctionComponent<{ history: History<unknown> }> = ({ history }) => {
   return (
-    <EuiErrorBoundary>
```

Comment:
`EuiErrorBoundary` is removed because the `KibanaErrorBoundary` is built-in to KibanaRenderContextProvider, and is preferred because of its integration with analytics. There were other opportunities to remove EuiErrorBoundary in this PR, but the focus of the Epic being worked on here is only to achieve dark mode consistency.